packages/escape-html/src/index.js

```diff
@@ -59,12 +70,15 @@ export function escapeLessThan( value ) {
  * "[...] the text cannot contain an ambiguous ampersand [...] must not contain
  * any literal U+0022 QUOTATION MARK characters (")"
  *
+ * Note we also escape the greater than symbol, as this is used by wptexturize to
+ * split HTML strings. This is a WordPress specific fix
+ *
```

Comment:
I think it would also be good to include an actionable note for the future maintainer. Something like:

```suggestion
 *
 * Note that if a resolution for Trac#45387 comes to fruition, it is no longer
 * necessary for `escapeGreaterThan` to exist in this module.
 *
 * See: https://core.trac.wordpress.org/ticket/45387
```